install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -427,7 +427,7 @@ spec:
         chart: {{ template "mixer.chart" $ }}
         heritage: {{ $.Release.Service }}
         release: {{ $.Release.Name }}
-        security.istio.io/mtlsReady: "true"
+        security.istio.io/tlsMode: "istio"
```

Comment:
are you sure I can remove this? I am simply renaming labels and not removing them, so that I dont get caught up in another issue. If this is sure to work, then I can remove the label

Reply:
up to you. i have another PR in flight to remove this anyway.

Reply:
@incfly the issue arises when `--set global.controlPlaneSecurityEnabled=true` and automtls is enabled.  Please make sure this is tested in your other PR